client/my-sites/marketplace/pages/marketplace-test/index.tsx

```diff
@@ -1,4 +1,6 @@
-import { YOAST_PREMIUM, YOAST_FREE, WOO_UPS_SHIPPING } from '@automattic/calypso-products';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: File used only for development and testing.
```

Comment:
I think you probably meant `// @ts-nocheck` (it also won't work with the trailing colon). I've fixed that in https://github.com/Automattic/wp-calypso/pull/59545

You might still want to fix the TS errors in this file. It has almost more than any other file. If you don't want to use TS, why not just make this a JS file?

Reply:
I preferred `@ts-ignore` so that I add an explanation. Anyhow, there isn't any point having this file as `ts` https://github.com/Automattic/wp-calypso/pull/59551